Project-II-Calculator/calculator/src/App.css

```diff
@@ -0,0 +1,51 @@
+.calculator{
```

Comment:
I appreciate how you used this to create more "global" stylings and then got more specific on each component.